rest/servers/spring/src/main/java/com/connexta/transformation/rest/spring/TransformApiDelete.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformation.rest.spring;
+
+import com.connexta.transformation.rest.models.ErrorResponse;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+/**
+ * Implementations of the {@code TransformApiDelete} interface provide a mechanism to implement the
+ * rest DELETE endpoint. This interface was manually separated out from the generated
+ * TransformApi.java file and any changes must be updated here as well. This interface also adds the
+ * throws Exception to the method signature.
```

Comment:
✏️ I would just indicate it is a split of the generated `TransformApi.java` to allow creation of separate controller classes along with what was changed/added. (same for all others).